pan_controller_5:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: pan_joint
    joint_speed: 0.3
    motor:
        id: 5
        init: 512
        min: 0
        max: 1023

tilt_controller_5:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: tilt_joint
    joint_speed: 0.3
    motor:
        id: 5
        init: 512
        min: 0
        max: 1023
